packages/core/src/resolvers/object.ts

```diff
@@ -15,6 +17,34 @@ export const resolveObject = ({
   combined?: boolean;
   context: ContextSpecs;
 }): ResolverValue => {
+  if (typeof schema === 'object' && (schema as SchemaObject)?.allOf) {
```

Comment:
is it causing a breaking change in the generated files?

Reply:
I think `Change's before after` is easy to understand.
https://github.com/anymaniax/orval/pull/861#issue-1713057019

`allOf` works as a mix in and is different from Typescript's intersection type. Therefore, it was necessary to merge the schema first.

Reply:
my bad didn't check all the comment of the pr